test/support/active_record.rb

```diff
@@ -1,3 +1,4 @@
+require "logger"
```

Comment:
I don't think we need this. I assume you used it for troubleshooting?

Reply:
This is the fix for the previous CI failures actually. This is because the latest concurrent-ruby doesn't require it anymore.

Reply:
Ah righty, yes I remember that from my other upgrades :) Sorry about that.